yos/copy.go

```diff
@@ -0,0 +1,133 @@
+package yos
+
+import (
+	"errors"
+	"io"
+	"math/bits"
+	"os"
+	"path/filepath"
+
+	"github.com/1set/gut/ystring"
+)
+
+var (
+	ErrShortRead  = errors.New("short read")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported var ErrShortRead should have comment or be unexported](https://app.codacy.com/app/an9an63/gut/pullRequest?prid=4727006)